fix: require explicit Linear persistence selection (#582)
howarewoo/monorepo-template: skills/woostack-plan/SKILL.md

---
name: woostack-plan

> description: Turn one approved specification into a complete PR-sized dependency-aware plan. Standalone planning may persist an explicitly selected Linear hierarchy; build-delegated planning returns a candidate graph without provider mutation. Never executes, commits, or merges.

Author: **Move workflow detail out of the description**

The changed frontmatter description embeds standalone-versus-delegated persistence behavior that is already documented in the skill body, violating the rule that workflow belongs in referenced documentation.

Fix: Reduce the frontmatter description to a concise discovery summary and leave mode-specific persistence behavior in the body and references.

```suggestion
description: Turn one approved specification into a complete PR-sized implementation plan with dependency-aware increments. Never executes, commits, or merges.
```

<sub>— <strong>HIGH · BLOCKING</strong> · <code>conventions</code></sub>

Author: Fixed in 059398c. The woostack-plan discovery description is concise again; mode-specific persistence behavior remains in the skill body and canonical references. The test parses the frontmatter key rather than depending on a physical line.
